lib/glimesh/streams.ex

```diff
@@ -108,4 +108,58 @@ defmodule Glimesh.Streams do
   def count_following(user) do
     Repo.one!(from f in Followers, select: count(f.id), where: f.user_id == ^user.id)
   end
+
+  alias Glimesh.Streams.StreamMetadata
+
+  def change_title(streamer, title) do
```

Comment:
`change_` is for generating a unsaved changeset like you have here: https://github.com/Glimesh/glimesh.tv/pull/87/files#diff-677844d838361a57c68f062b05853dd6R48

`update_` is for actually committing the transaction. Checkout accounts.ex `change_user_profile` vs `update_user_profile`